package.json

```diff
@@ -26,6 +26,9 @@
     "lint:js": "eslint --cache .",
     "lint:spelling": "cspell \"**/*.*\"",
     "lint": "npm-run-all -l -p \"lint:**\"",
+    "fix:js": "npm run lint:js -- --fix",
+    "fix:prettier": "npm run lint:prettier -- --write",
+    "fix": "npm-run-all -l -p \"fix:**\"",
```

Comment:
@snitin315 We can't run them using `-p`, because eslint and prettier start racing, firstly we need to run eslint and then prettier

Reply:
Oh! Yes. @ersachin3112 can you update accordingly?